cmd/sebak/cmd/run.go

```diff
@@ -87,8 +87,11 @@ func init() {
 
 			parseFlagsNode()
 
-			runNode()
-			return
+			var exitCode int
+			if err = runNode(); err != nil {
+				exitCode = 1
+			}
+			os.Exit(exitCode)
```

Comment:
```go
if err = runNode(); err != nil {
  os.Exit(1)
}
os.Exit(0)

Reply:
@Geod24 I want to open the possibility to set the another exit code like http://www.tldp.org/LDP/abs/html/exitcodes.html .

Reply:
Then the exit code should be returned from the function and used directly, no ?

Reply:
@Geod24 The way to set the exit code, we can discuss. Checking error type, or `runNode()` return the exit code explicitly instead of error. Anything can be possible. :)

Reply:
You're right, it's not a big deal, let's leave it as is